README.md

```diff
@@ -231,18 +231,35 @@ Message filters:
 
   - AllowMsgIdOut: If set, only allow messages with the listed message IDs to
     be sent via this endpoint
+  - BlockMsgIdOut: If set, block messages with the listed message IDs to
+    be sent via this endpoint
```

Comment:
Lines should be max. 80 characters wide, so the `be sent` shall still be on the first line.